### PR TITLE
Scripts: Added script for Orcish Counterstance for those WotG Orcs.

### PR DESCRIPTION
**Pull request by TeoTwawki**

### CODE REVIEW

File: scripts/globals/mobskills/Orcish_Counterstance.lua [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+-- Orcish Counterstance
+-- Used only by Orcs in Wings of the Goddess Areas.
+-- Certain NMs may have a higher power version of the effect.
+---------------------------------------------
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    if (mob:getContinentID() == 3) then -- THE_SHADOWREIGN_ERA = 3
```
Comment (xdemolish) [inline scripts/globals/mobskills/Orcish_Counterstance.lua]: stick the continent ids in status.lua since it's required anyways

**Comment by TeoTwawki** [comment]
Updated, but placed IDs in zone.lua instead of status.lua per discussion on IRC ("demo told me to"). :grinning:

File: scripts/globals/zone.lua [hunk 2]
```diff
@@ -1,3 +1,80 @@
...
+------------------------------------
+
+    -- Zone Types
+    NONE        = 0
+    CITY        = 1
+    OUTDOORS    = 2
+    DUNGEON     = 3
+    BATTLEFIELD = 4
+    DYNAMIS     = 5
+    INSTANCED   = 6
```
Comment (xdemolish) [inline scripts/globals/zone.lua]: ZONETYPE_CRAP

File: scripts/globals/zone.lua [hunk 3]
```diff
@@ -1,3 +1,80 @@
...
+    ARAGONEAU_FRONT  = 38
+    FAUREGANDI_FRONT = 39
+    VALDEAUNIA_FRONT = 40
+    ABYSSEA          = 41
+    THE_THRESHOLD    = 42
+    ABDHALJS         = 43
+    ADOULIN_ISLANDS  = 44
+    EAST_ULBUKA      = 45
+
+    UNKNOWN          = 255
```
Comment (xdemolish) [inline scripts/globals/zone.lua]: REGION_TYPE

File: scripts/globals/zone.lua (hunk 2 above)
Comment (TeoTwawki) [inline scripts/globals/zone.lua]: Its not required that they perfectly match whats in the source header file. Only that the use be constant in all lua script. I don't understand why the souce is usinging a prefix all those enums, when what they are is plain as day without any prefixing of the individual things
```cpp
enum ZONETYPE
{
    ZONETYPE_stuff = number,
}
```
is there an actual purpose in those prefixes? I normally like my lua stuff to match my core stuff...but these ones annoy me the same way a comment net to the enum telling me "enums zonetypes" would.

File: scripts/globals/zone.lua (hunk 3 above)
Comment (TeoTwawki) [inline scripts/globals/zone.lua]: F it, I'll make'em match. But it annoys me and I will have you all here of it! /ramble /gripe /complain

**Comment by teschnei** [comment]
the purpose is making global namespace conflicts more difficult, since they are not encased in a table

**Comment by TeoTwawki** [comment]
Its amended to include the prefixes, but juts sayin: those names are extremely likely to have said conflicts. But I guess consistency in formatting is important enough too and I'd be hypocrite if I argued it.

**Pull request merged**